js/popup.js

```diff
@@ -0,0 +1,281 @@
+    },
+  },
+
+  {
+    title: "Uber Navigation",
+    roles: { company: "Uber", position: ["Lead Developer", "2018"] },
+    image: "multi-stories-post.svg",
+    alt: "Uber Navigation project screenshot",
+    description:
+      "A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car..",
+    skills: ["html", "Ruby on rails", "css", "javascript"],
+    skillsModal: ["html", "css", "javascript", "github", "ruby", "Bootstraps"],
+    livelink: { link: "#", text: "See live" },
+    seesource: { link: "#", text: "See Source" },
+    seeproject: "See Project",
+    htmlClass: {
+      article: [
+        "grid-item",
+        "card",
+        "flex-item-3",
+        "flex-item-last",
+        "hide-ruby",
+      ],
+      cardImage: ["card-image", "flex-item-2"],
+      cardDescription: ["card-description", "flex-item-1"],
+    },
+  },
+];
+
+const dummyText =
+  "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.";
+const mobileText =
+  "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent";
+
+const listItems = (items) => {
+  let list = "";
+  items.forEach((element) => {
+    list += `<li>${element}</li>`;
+  });
+
+  return list;
+};
+
+const createImageButton = (project, dest, srcLink) => {
+  const link = document.createElement("a");
+  link.className = "btn";
+  link.textContent = `${dest.text} `;
+  link.href = dest.link;
+  const img = document.createElement("img");
+  img.src = srcLink;
+  img.alt = `${dest.text} Icon`;
+  link.appendChild(img);
+
+  return link;
+};
+
+const modalFooter = (project) => {
+  const modalButtons = document.createElement("div");
+  modalButtons.className = "show";
+  const seeSourceButton = createImageButton(
+    project,
+    project.livelink,
+    "Icon.svg"
+  );
+  modalButtons.appendChild(seeSourceButton);
+  const seeLiveButton = createImageButton(
+    project,
+    project.seesource,
+    "sourcecode.svg"
+  );
```

Comment:
- [ ] The modal buttons seem to be stacked on top of each other instead of being on the same row as shown below. Please follow the design guidelines 👇 
- [ ] Please add the horizontal bottom bar in the mobile view pop up
### Your Design
![image](https://user-images.githubusercontent.com/46839548/168441955-3ca85ff1-8131-4724-8553-16205a2d09f1.png)

### Figma Desing
![image](https://user-images.githubusercontent.com/46839548/168441994-fe774285-804e-4fb5-8d32-d0d918683255.png)